client/state/posts/utils.js

```diff
@@ -177,8 +177,8 @@ export function getTermIdsFromEdits( post ) {
 		return post;
 	}
 
-	// Skip "default" taxonomies until legacy token-field and category-selector are removed
-	let taxonomies = omit( post.terms, [ 'post_tag', 'category' ] );
+	// Skip "post_tag" taxonomies until legacy token-field is removed
+	let taxonomies = omit( post.terms, [ 'post_tag' ] );
```

Comment:
Aside: If we moved the `post_tag` omit check into the `reduce` below as a condition, it'd save a few cycles:

```js
const taxonomies = reduce( post.terms, ( prev, taxonomyTerms, taxonomyName ) => {
	if ( 'post_tag' === taxonomyName ) {
		return prev;
	}

	// Ensures we are working with an array
	const termsArray = toArray( taxonomyTerms );
	if ( termsArray && termsArray.length && ! isPlainObject( termsArray[ 0 ] ) ) {
		return prev;
	}

	prev[ taxonomyName ] = termsArray;
	return prev;
}, {} );
```